examples/rest.rs

```diff
@@ -17,7 +17,8 @@ pub fn main() {
         .with_body(r#"{"username":"{USER}","password":"{PASS}"}"#.to_string())
         .with_selector("/status".to_string())
         .with_regex("pass".to_string())
-        .build();
+        .build()
+        .unwrap();
```

Comment:
I would like to remove the unwraps from the examples too.

Reply:
done